micromamba/tests/test_create.py

```diff
@@ -143,6 +143,20 @@ def test_env_lockfile_different_install_after_create(tmp_home, tmp_root_prefix,
     helpers.install("-p", env_prefix, "-f", install_spec_file, "-y", "--json")
 
 
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+def test_env_spdlog_overhead_regression(tmp_home, tmp_root_prefix, tmp_path):
```

Comment:
xeus-cling is not available on Windows and I guess the test on OSX is actually done for ARM64 arch, for which xeus-cling is not available either. We should probably limit this test to Linux only.